2.5/test/run

```diff
@@ -161,17 +161,6 @@ function test_base_scl_usage() {
   check_result "$?"
 }
 
-function test_application() {
```

Comment:
Just to be sure- is the intetion to prefer the shorter function, without the `test_connection`?

https://github.com/sclorg/s2i-ruby-container/pull/288/files/f01d4b526f824613db8d46c63bae1d6715a8c558..cb4f86209ded75fe100086fb99195ff2c86baa5b#diff-6c238a2939d34959bb9e593a55eebf6f805fe96b556ea90f98eb53d7a53e10faL201-L207

Reply:
Yes. I have separated them into two test cases.